integ/src/ec2_provider.rs

```diff
@@ -138,6 +145,30 @@ pub async fn terminate_ec2_instance(cluster: ClusterInfo) -> ProviderResult<()>
     Ok(())
 }
 
+// Find the node ami id to use.
+async fn find_ami_id(
+    ssm_client: &aws_sdk_ssm::Client,
+    arch: &str,
+    br_version: &str,
+    eks_version: &str,
+) -> ProviderResult<String> {
+    let parameter_name = format!(
+        "/aws/service/bottlerocket/aws-k8s-{}/{}/{}/image_id",
```

Comment:
Maybe define this as a pattern variable?

Reply:
Can you give an example about it?  thanks

Reply:
I don't think we can use `format!(PARAMETER_PATERN, ...)` since format! needs syntax like format!({}, ...)`. We can change it to 
```
const PARAMETER_PATERN = "/aws/service/bottlerocket/aws-k8s-"
let parameter_name = format!({}{}/{}/{}/image_id, PARAMETER_PATERN, ...)
```
But I don't think this is readable, so I would keep current format. Thanks. 👍